lib/repo/git_hooks/client/pre-commit.d/04-file_size_too_large.py

```diff
@@ -16,7 +19,7 @@
         file_size_proc = subprocess.run(['git', 'cat-file', '-s', file_sha],
                                    stdout=subprocess.PIPE, universal_newlines=True)
         file_size = file_size_proc.stdout.strip()
-        if int(file_size) > MAX_FILE_SIZE:
```

Comment:
You can delete the `MAX_FILE_SIZE` constant as well (line 4), similar to the server-side hook.